internal/chrome_desktop.py

```diff
@@ -32,16 +32,19 @@
     '--disable-device-discovery-notifications',
```

Comment:
this flag is gone now and replaced (AFAIUI) by `--disable-features=OptimizationHints`